Sources/NIOExtras/PCAPRingBuffer.swift

```diff
@@ -13,17 +13,19 @@
 //===----------------------------------------------------------------------===//
 
 import NIOCore
+import NIOConcurrencyHelpers
 
 // MARK: NIOPCAPRingBuffer
 /// Storage for the most recent set of packets captured subject to constraints.
 /// Use `addFragment` as the sink to a `NIOWritePCAPHandler` and call `emitPCAP`
 /// when you wish to get the recorded data.
 /// - Warning:  This class is not thread safe so should only be called from one thread.
-public class NIOPCAPRingBuffer {
+public class NIOPCAPRingBuffer: @unchecked Sendable {
```

Comment:
Does this class actually need to be `Sendable`? The warning above indicates it isn't thread safe and should be used appropriately (i.e. the caller is responsible for ensuring thread safety)

Reply:
I guess now it _is_ threadsafe we can remove the warning?

Reply:
Right, my point is that being sendable isn't always desirable. 

I just want to make sure we're adding it for the right reason (because we think it _should_ be sendable rather than for the sake of making it sendable) given it was intentionally not made thread-safe previously.

Reply:
Sorted :)